domains/arabiancoconut.is-cool.dev.json

```diff
@@ -0,0 +1,20 @@
+{
+    "schema": "../schemas/domain.schema.json",
+    "description": "The domain for project use",
+    "domain": "is-cool.dev",
+    "subdomain":["arabiancoconut","_github-pages-challenge-ArabianCoconut.arabiancoconut.is-cool.dev"],
+
+    "owner":{
+        "repo":"https://github.com/ArabianCoconut",
+        "email":"p11h0ifd@duck.com",
+        "website":"https://arabiancoconut.github.io"
+    },
+
+    
+    "record":{
+        "TXT":["69dddce1d886bf46fce9aa94529895"],
+        "CNAME":["arabiancoconut.github.io."],
+        "A":["185.199.108.153","185.199.109.153","185.199.110.153 ","185.199.111.153"]
```

Comment:
You can't have a CNAME record and also have other records. CNAME must stand by itself at all times.

https://www.rfc-editor.org/rfc/rfc1034#section-3.6.2